src/vmm/benches/cpu_templates.rs

```diff
@@ -0,0 +1,34 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Benchmarking cases:
+//   * `CustomCpuTemplate` Deserialization
+
+use std::path::Path;
+
+use criterion::{criterion_group, criterion_main, Criterion};
+use vmm::guest_config::templates::test_utils::TEST_TEMPLATE_JSON;
```

Comment:
The test json templates are rather short. I guess their size reflects a typical customer template size, but don't we want to test bigger templates to give us some margin of confidence?

Reply:
I do want to expand these test for sure. Right now, I just want the benchmark to protect against changes that introduce order of magnitude regressions in the future. But we can and will be doing more.

Reply:
That's fair enough.